src/components/Toggle/Toggle.stories.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React, { Component, Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import Showcase from '../../../.storybook/components/Showcase';
+import Toggle from './Toggle';
+
+class StatefulToggle extends Component<any, any> {
+  state = {
+    isToggled: true,
+  };
+
+  toggle = () => {
+    this.setState(state => ({ isToggled: !state.isToggled }));
+  };
+
+  render() {
+    return (
+      <Toggle
+        onToggle={this.toggle}
+        isToggled={this.state.isToggled}
+        {...this.props}
+      />
+    );
+  }
+}
+
+storiesOf('Toggle', module).add(
```

Comment:
Yeah, so I brought it up in chat, but I think we should switch to Docz. It would make it much easier / more conventional to do stuff like this, documenting the props that a component takes and showing it in action.

I think I'm gonna spend some time converting our existing stories, to see how it feels.